airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/config.py

```diff
@@ -189,6 +189,9 @@ class TestStrictnessLevel(str, Enum):
         high = "high"
         low = "low"
 
+    cache_discovered_catalog: bool = Field(
+        default=True, description="Enable or disable caching of discovered catalog for reuse in multiple tests."
+    )
```

Comment:
Should the default be false to prevent these kind of errors?  What's the slow-down introduced if we don't cache?

Reply:
If we don't cache the discovered catalog, every test using it will trigger a `discover` command run as the scope of the `discovered_catalog` fixture is `function` by default. As this caching feature did not bother us until now, a specific edge case, I would suggest keeping it as a default. 
We could try to make it smarter and leverage another fixture scope  for more granular caching (e.g. `class` scope).

Reply:
👍